.github/ISSUE_TEMPLATE/--first-timers-only.md

```diff
@@ -53,7 +52,7 @@ $DIFF
 
 2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.
 
-- [ ] 🏁 **Done** Ask in comments for a review :)
+- [ ] 🏁 **Done** Ask in the comments for a review :)
```

Comment:
Thanks for adding this @ojninja16!

Reply:
@TildaDares I added the issue link above .. Is it ok now??

Reply:
> Thanks for adding this @ojninja16!

Thank you for giving me this opportunity